config/dev/eks-managedcluster.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: ManagedCluster
+metadata:
+  name: eks-dev-slava-1
+  namespace: ${NAMESPACE}
+spec:
+  config:
+    publicIP: true
```

Comment:
Was it `false` (I don't see the previous revision)? I don't see the reason to use public IPs here